### PR TITLE
Add support for ticket cancellation

### LINKED ISSUE #18: Add cancel functionality to 'previous complaints' view
Add a cancel button with the previous complaints and requests for the user to cancel their previous, open complaints.

### PR DESCRIPTION
**Pull request by Kushagra-0801**
Added two new url endpoints for ticket cancellation
Closes #18

### CODE REVIEW

**Comment by iamkroot** [comment]
The URL endpoints are not proper. URLs should follow a `/<model>s/<pk>/<verb>` format. For example: `/complaints/12/cancel`.

**Comment by Kushagra-0801** [comment]
@iamkroot fixed the urls

File: complaints/urls.py [hunk 1]
```diff
@@ -13,4 +13,8 @@
         views.handle_unblock_request,
         name="update_request_resolve",
     ),
+    path("complaint/<int:pk>/cancel/", views.cancel_complaint, name="cancel-complaint"),
```
Comment (iamkroot) [inline complaints/urls.py]: Plural please. `complaints`

File: complaints/views.py [hunk 2]
```diff
@@ -173,7 +203,7 @@ def email_on_request(request_id, category, details, issue, user_email):
 
 
 def email_resolve(
-        request_id, request_status, issue, user_email, category, details, remark_user
+    request_id, request_status, issue, user_email, category, details, remark_user
 ):
     """Function to send email to the user when his request or compliant gets resolved"""
     subject = f"{issue} request_status"
```
Comment (Aviral14) [inline complaints/views.py]: put {request_status} instead of request_status

File: complaints/urls.py (hunk 1 above)
Comment (Kushagra-0801) [reply complaints/urls.py]: done

File: complaints/views.py (hunk 2 above)
Comment (Kushagra-0801) [reply complaints/views.py]: done

**Comment by Aviral14** [comment]
The commit messages are not proper.what is made plural? Explain that.Moreover, explain properly what your second commit does.Don't type anything gibberish in commit messages jusk for the sake of writing something.

**Comment by iamkroot** [comment]
Please improve previous commit's message body. Be more specific as to what is being fixed/added/chamged.

**Comment by Kushagra-0801** [comment]
@iamkroot Changed the commit message body. Please review and merge

**Comment by iamkroot** [comment]
Please rebase with master locally and squash the latest commit into its previous one. There should be two commits in this PR.

**Pull request merged**